config/locales/doorkeeper.fa.yml

```diff
@@ -3,8 +3,10 @@ fa:
   activerecord:
     attributes:
       doorkeeper/application:
-        name: Name
+        name: Application name
```

Comment:
work in progress?

Reply:
I intended it to be final. I just copied whatever I found in the corresponding English file. I saw a couple of words in `config/locales/doorkeeper.en.yml` had changed and I reflected those in the Persian translation.

Reply:
i see.

Reply:
Oh you mean why aren't they translated? That was a conscious decision from the beginning to not translate that file for the moment because (a) it is mostly admin-related and (b) there is still no agreed-upon Persian translation for most of those terms and a translation would only confuse admins.